build/NuGet.targets

```diff
@@ -25,74 +22,40 @@
 
 	<PropertyGroup>
 		<NuGetToolsPath>$(MSBuildThisFileDirectory)</NuGetToolsPath>
-		<PackagesConfig Condition="'$(OS)'=='Windows_NT'">$(ProjectDir)packages.config</PackagesConfig>
-		<PackagesConfig Condition="'$(OS)'!='Windows_NT'">$(ProjectDir)/packages.config</PackagesConfig>
```

Comment:
This change doesn't seem related...I suppose you are quite confident that the leading slash for linux is (no longer?) needed?